tsl/src/remote/connection.c

```diff
@@ -326,6 +328,8 @@ fill_result_error(TSConnectionError *err, int errcode, const char *errmsg, const
 	err->remote.stmtpos = get_error_field_copy(res, PG_DIAG_STATEMENT_POSITION);
 	if (err->remote.msg == NULL)
 		err->remote.msg = pstrdup(PQresultErrorMessage(res));
+	if (err->remote.msg == NULL)
+		err->remote.msg = pstrdup(PQerrorMessage(entry->conn->pg_conn));
```

Comment:
This won't work, should either change to length check or remove. I think this was needed for error reporting when the data node segfaults.